src/NServiceBus.Hosting.Azure/DynamicHost/DynamicHostControllerConfig.cs

```
@@ -1,56 +1,90 @@
 namespace NServiceBus.Hosting
 {
+    using System;
     using System.Configuration;
+    using NServiceBus.Features;
 
+    [ObsoleteEx(Message = "The profiles configured via AzureProfileConfig are no longer supported",
```

Same question - if we obsolete this, shouldn't we either throw an exception or drop the code?